src/tests/JIT/opt/Vectorization/UnrollEqualsStartsWIth_minopts.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <Optimize>True</Optimize>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="UnrollEqualsStartsWIth.cs" />
+    <Compile Include="UnrollEqualsStartsWIth_Tests.cs" />
```

Comment:
NOTE: these tests have test cases for Equals(OrdinalIgnoreCase) but mostly for optimized code where jit unrolls/vectorize them. I'm adding a minopts version to run them without jit's foldings